src/resources/i18n/translations/en.json

```diff
@@ -47,7 +47,7 @@
   "Are you sure you want to delete all of your downloaded episodes from this podcast": "Delete all downloaded episodes from this podcast?",
   "Are you sure you want to delete your account": "Delete your account?",
   "Ask to sync local podcasts with server": "Save the local subscriptions to Podverse Premium so they are available on all devices?",
-  "Auto queue new episodes position": "Position to auto-enqueue episodes",
+  "Auto queue new episodes position": "Auto-enqueue episodes …",
```

Comment:
```suggestion
  "Auto queue new episodes position": "Auto add episodes …",
```

Reply:
This feature only auto-adds new episodes as they're detected in feeds, not clips.

Since it's on the Queue screen how about "Auto-add new episodes". I think "new" is important to specify which episodes will get auto-added.

Reply:
It also doesn't delete episodes from the queue once not new(?)

Reply:
@mitchdowney Good idea, but is it only for downloaded episodes?
"Add downloaded episodes" would suffice.

Reply:
"Add downloaded episodes" works for me. Change made in this commit: https://github.com/podverse/podverse-rn/commit/74933d3b4cbb180ea672179cbb89db49ad180f7f